doc(readme): warn user about docker/lxd connectivity issue (#26)
canonical/colcon-in-container: README.md

sudo snap install lxd
lxd init --auto
```

> > :warning: If you have Docker installed, make sure to
> > [prevent connectivity issue with LXD and Docker](https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#prevent-connectivity-issues-with-lxd-and-docker)

Reviewer: ```suggestion
> :warning: If you have Docker installed,
> [mind that it causes connectivity issues](https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#prevent-connectivity-issues-with-lxd-and-docker)
```
